docker/docker-compose.1804.54.yaml

```diff
@@ -3,14 +3,16 @@ version: "3"
 services:
 
   runtime-setup:
-    image: swift-distributed-tracing-baggage:18.04-5.4
+    image: swift-distributed-tracing-baggage:20.04-5.4
```

Comment:
Shouldn't this stay `18.04` as the file name suggests or be renamed to `20.04`?

Reply:
Gotta move to 20 I think, will fix filename